src/main/java/seedu/address/model/meeting/Meeting.java

```diff
@@ -76,15 +86,14 @@ public boolean isSameMeeting(Meeting otherMeeting) {
             return true;
         }
 
-        return otherMeeting != null
-                && otherMeeting.getTitle().equals(getTitle())
+        return otherMeeting != null && otherMeeting.getTitle().equals(getTitle())
                 && otherMeeting.getLocation().equals(getLocation())
-                && otherMeeting.getMeetingTime().equals(getMeetingTime());
+                && otherMeeting.getMeetingTime().equals(getMeetingTime()) && otherMeeting.getTags().equals(getTags());
```

Comment:
I don't think that meeting tags should be included in this. Because this means that 2 meetings with the same name, time and location can happen at the same time and are not duplicates. This makes it confusing as you may add a tag to the wrong meeting if you have duplicates or create a new meeting with the exact same details but without the tags